src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -113,3 +116,5 @@ implication        ,ARM64 ,Dp        ,AdvSimd
 implication        ,ARM64 ,Rdm       ,AdvSimd
 implication        ,ARM64 ,Sha1      ,ArmBase
 implication        ,ARM64 ,Sha256    ,ArmBase
+implication        ,ARM64 ,Vector64  ,AdvSimd
+implication        ,ARM64 ,Vector128 ,AdvSimd
```

Comment:
Same for ARM64. Just ensuring Vector64/128 are correctly marked as being dependent on AdvSimd.